doc/batcher-api.md

```diff
@@ -0,0 +1,143 @@
+# Batcher API Guide
+
+The batcher has two responsabilities: choosing which aggregator to choose for a metric instrument and store the last record for each metric ready to be exported.
```

Comment:
```suggestion
The batcher has two responsibilities: choosing which aggregator to choose for a metric instrument and store the last record for each metric ready to be exported.
```

Reply:
Pls add link to Batcher class: https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-metrics/src/export/Batcher.ts